### PR TITLE
Series view

### LINKED ISSUE #16: series view
Same requirements as character view, for now

### PR DESCRIPTION
**Pull request by vacas5**
Not as many links to show with series. In fact I am not even sure what
series are, but it at least gives us another view.

Fixes #16

### CODE REVIEW

**Comment by vacas5** [comment]
I will actually want to move in the logic from my other PR before merging this one, but this gets us started

File: src/SeriesList.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import AlphaLinks from './AlphaLinks';
+import Loader from './Loader';
+import Card from './Card';
+
+class SeriesList extends Component {
```
Comment (nicrocs) [inline src/SeriesList.js]: I think we could create a List container component that could at lease be responsible for the fetching of both the Character and the Series lists.

**Comment by nicrocs** [comment]
If you want to keep the Series and Character List components as they are for now, that will totally work, but I think we should consider creating a container component that will do the fetching and render the Series or Character List depending on props.

**Comment by vacas5** [comment]
yep I think that would be a good step

**Pull request merged**